### PR TITLE
chore(wiki): Remove deprecated files

### PR DESCRIPTION
**Pull request by Kitzunu**

### CODE REVIEW

File: _includes/azerothcore/sidebar.html [hunk 1]
```diff
@@ -49,9 +49,4 @@
         <li><a href="/wiki/Waypoints-Information">Information about waypoints</a></li>
         <li><a href="/wiki/How-to-obtain-free-records">How to obtain free records</a></li>
     </ul>
-    <b>Archive</b>
-    <ul>
-        <li><a target="_blank" href="http://www.azerothcore.org/pages/sunwell.pl/">Original SC Description</a></li>
```
Comment (Helias) [inline _includes/azerothcore/sidebar.html:54]: why we should remove this? I would keep it

Comment (Kitzunu) [reply _includes/azerothcore/sidebar.html:54]: I remove it because it serves little to no purpose

Comment (Helias) [reply _includes/azerothcore/sidebar.html:54]: for me it's useful

Comment (FrancescoBorzi) [reply _includes/azerothcore/sidebar.html:54]: I agree with @Helias

**Review by FrancescoBorzi — APPROVED** [review]

**Pull request merged**